src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Figure figure;

```

Comment:
```suggestion
```

Reply:
I can not delete this, because i use this for creating other figures

Reply:
Create it inside method